src/app/public/modules/media-query/media-query.service.ts

```diff
@@ -78,6 +106,9 @@ export class SkyMediaQueryService implements OnDestroy {
     });
   }
 
+  /**
+   * @internal
+   */
   public destroy(): void {
```

Comment:
This method still shows in the docs. File issued here: https://github.com/blackbaud/skyux-docs-tools/issues/108